client/apps/admin/components/common/sortable.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Sortable(props) {
+  const {
+    title,
+    column,
+    currentColumn,
```

Comment:
```suggestion
    currentSortColumn,
```

Reply:
I think because this is the sortable componant, naming the prop with Sort in there is too redundant. You?